lib/ansible/modules/system/puppet.py

```diff
@@ -260,6 +266,11 @@ def main():
             cmd += " --certname='%s'" % p['certname']
         if module.check_mode:
             cmd += " --noop"
+        if p['use_srv_records']:
```

Comment:
this should be validaidating to non None:

~~~python
if p['use_srv_records'] is not None:
~~~